app/controllers/admin/users_controller.rb

```diff
@@ -13,6 +13,17 @@ def index
       end
     end
 
+    def edit
+      @user = visible_users.find_by(id: params[:id])
```

Comment:
This can be simplified to `@user = record`, since the `id` param is automatically used to initialize a special `record` variable. Same below (and possibly in the courses controller---I didn't think of this when reviewing that earlier PR).

Reply:
The `record` variable was used in the courses controller. I just thought it was better style here to select from all non autotest users (as opposed to from all Users which is what `record` queries from). Not a big deal either way though. Will make this change.